server/src/models/world/Spawner.ts

```diff
@@ -361,7 +361,7 @@ export class Spawner {
 
   public getRandomPath(): string {
     if (!this.hasPaths) return '';
-    return sample(this.paths);
+    return sample(this.paths) ?? '';
```

Comment:
this function could return string | undefined I believe instead

Reply:
Nope, that didn't work

Reply:
Alternative didn't work